packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -242,6 +242,12 @@ export function getDevMiniflareOptions(
 			],
 			bindings: {
 				CONFIG: {
+					...(entryWorkerConfig?.compatibility_date
+						? { compatibility_date: entryWorkerConfig?.compatibility_date }
+						: {}),
+					...(entryWorkerConfig?.compatibility_flags
+						? { compatibility_flags: entryWorkerConfig.compatibility_flags }
+						: {}),
```

Comment:
These will only be added if there is a User Worker. I think that's correct because this behaviour is not relevant with assets only but just wanted to check.

Reply:
Nope, you're totally right. I've picked up these options from config now whether or not there is an user Worker.